src/build/common/tasks/publish.yaml

```diff
@@ -1,12 +1,16 @@
+# Task to publish a path as an artifact
 parameters:
-  path: ''
-  artifactName: '$(Build.DefinitionName)'
-  displayName: 'Artifact Publish'
+  - name: path
+    default: ''
+  - name: artifactName
+    default: '$(Build.DefinitionName)'
+  - name: displayName
+    default: 'Artifact Publish'
 
 steps:
   - task: PublishBuildArtifacts@1
     displayName: ${{parameters.displayName}}
     inputs:
       PathtoPublish: '$(build.ArtifactStagingDirectory)/${{parameters.path}}'
       ArtifactName: ${{parameters.artifactName}}
-      condition: and(succeeded(), and(not(eq(variables['Build.Reason'], 'PullRequest')), not(eq(variables['Build.Reason'], 'Schedule'))))
\ No newline at end of file
+    condition: and(succeeded(), and(not(eq(variables['Build.Reason'], 'PullRequest')), not(eq(variables['Build.Reason'], 'Schedule'))))
```

Comment:
question: why would we not want to do this if it were a schedule? Seems reasonable to me to do this

Reply:
Agreed, have removed.